src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -220,46 +270,121 @@ public static class Sse2
         /// </summary>
         public static Vector128<double> CompareLessThan(Vector128<double> left, Vector128<double> right) => CompareLessThan(left, right);
 
+        /// <summary>
+        /// int _mm_comilt_sd (__m128d a, __m128d b)
+        /// </summary>
+        public static bool CompareLessThanOrderedScalar(Vector128<double> left, Vector128<double> right) => CompareLessThanOrderedScalar(left, right);
+
+        /// <summary>
+        /// int _mm_ucomilt_sd (__m128d a, __m128d b)
+        /// </summary>
+        public static bool CompareLessThanUnorderedScalar(Vector128<double> left, Vector128<double> right) => CompareLessThanUnorderedScalar(left, right);
+
+        /// <summary>
+        /// __m128d _mm_cmplt_sd (__m128d a,  __m128d b)
+        /// </summary>
+        public static Vector128<double> CompareLessThanScalar(Vector128<double> left, Vector128<double> right) => CompareLessThanScalar(left, right);
+
         /// <summary>
         /// __m128d _mm_cmple_pd (__m128d a,  __m128d b)
         /// </summary>
         public static Vector128<double> CompareLessThanOrEqual(Vector128<double> left, Vector128<double> right) => CompareLessThanOrEqual(left, right);
 
+        /// <summary>
+        /// int _mm_comile_sd (__m128d a, __m128d b)
+        /// </summary>
+        public static bool CompareLessThanOrEqualOrderedScalar(Vector128<double> left, Vector128<double> right) => CompareLessThanOrEqualOrderedScalar(left, right);
+
+        /// <summary>
+        /// int _mm_ucomile_sd (__m128d a, __m128d b)
+        /// </summary>
+        public static bool CompareLessThanOrEqualUnorderedScalar(Vector128<double> left, Vector128<double> right) => CompareLessThanOrEqualUnorderedScalar(left, right);
+
+        /// <summary>
+        /// __m128d _mm_cmple_sd (__m128d a,  __m128d b)
+        /// </summary>
+        public static Vector128<double> CompareLessThanOrEqualScalar(Vector128<double> left, Vector128<double> right) => CompareLessThanOrEqualScalar(left, right);
+
         /// <summary>
         /// __m128d _mm_cmpneq_pd (__m128d a,  __m128d b)
         /// </summary>
         public static Vector128<double> CompareNotEqual(Vector128<double> left, Vector128<double> right) => CompareNotEqual(left, right);
 
+        /// <summary>
+        /// int _mm_comine_sd (__m128d a, __m128d b)
```

Comment:
Would you like to fix this comment to `_mm_comineq_sd `?

Reply:
Fixed.